propel/propel3/2017-02-14.yaml

```diff
@@ -0,0 +1,8 @@
+title:     SQL injection possible with limit() on MySQL
+link:      https://github.com/propelorm/Propel3/issues/73
+cve:       ~
+branches:
+    "3.0":
+        time:     2018-02-16 13:38:00
+        versions: ['>=3', '<4']
+reference: composer://propel/propel3
```

Comment:
I think this file should be removed as the package does not exist on Packagist, so this file will never match any `composer.lock` file.

Reply:
Yes, v3 isn't released nor made public, so there's no point in tagging this version as vulnerable.